drivers/espi/host_subs_npcx.c

```diff
@@ -1057,13 +1067,11 @@ int npcx_host_init_subs_core_domain(const struct device *host_bus_dev,
 	if (IS_ENABLED(CONFIG_PM)) {
 		/*
 		 * Configure the host access wake-up event triggered from a host
-		 * transaction on eSPI/LPC bus. No need for callback function.
+		 * transaction on eSPI/LPC bus. Do not enable it here. Or plenty
+		 * of interrupts will jam the system in S0.
 		 */
 		npcx_miwu_interrupt_configure(&host_sub_cfg.host_acc_wui,
 				NPCX_MIWU_MODE_EDGE, NPCX_MIWU_TRIG_HIGH);
```

Comment:
Yes, it can. But the configuration for miwu input signal (rising edge) only needs to be done once. From an efficiency point of view, the driver needn't do it every time ec enters deep sleep.

Reply:
Hi @keith-zephyr :  
Any further comment?